YAFC/Utils/CommandLineParser.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.IO;
```

Comment:
Does it make sense to use [`System.CommandLine`](https://learn.microsoft.com/en-us/dotnet/standard/commandline/)? It is still in beta/testing, but I would assume it becomes a standard at some point?

Reply:
I thought about it, since it seems legit. But in the end, I decided to wait for it to become a standard, production-ready part of .Net.